schema/spacewalk/common/data/rhnPackageKey.sql

```diff
@@ -172,6 +172,9 @@ insert into rhnPackageKey (id, key_id, key_type_id, provider_id) (select sequenc
 -- Amazon Linux 2
 insert into rhnPackageKey (id, key_id, key_type_id, provider_id) (select sequence_nextval('rhn_pkey_id_seq'), '11cf1f95c87f5b1a', lookup_package_key_type('gpg'), lookup_package_provider('Amazon') from dual where not exists (select 1 from rhnPackageKey where key_id = '11cf1f95c87f5b1a'));
 
+-- Amazon Linux 2023
+insert into rhnPackageKey (id, key_id, key_type_id, provider_id) (select sequence_nextval('rhn_pkey_id_seq'), '2f7fe951904ad832c631', lookup_package_key_type('gpg'), lookup_package_provider('Amazon') from dual where not exists (select 1 from rhnPackageKey where key_id = '2f7fe951904ad832c631'));
```

Comment:
I wonder if it's normal that the key_id is longer here, did I do a mistake?

But if I did, how is that after applying the sql script from the migration the vendor is still being identified:

![imagen](https://github.com/uyuni-project/uyuni/assets/4226070/9b9996e7-80a9-4f63-bf0d-80bfab6b849e)

@mackdk, any thoughts?

Reply:
I honestly have no clue. Maybe @mcalmer can help?

Reply:
Interesting:
```
Signature   : RSA/SHA512, Thu Feb  2 03:21:21 2023, Key ID e951904ad832c631
```
And at the instance
```
bash-5.2# gpg /etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2023
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
pub   rsa4096 2022-12-08 [SC]
      B21C50FA44A99720EAA72F7FE951904AD832C631
uid           Amazon Linux <amazon-linux@amazon.com>
```

I wonder from where I even took that GPG key, I will fix this.

Bu it still, how is it possible that Uyuni is able to identify the Vendor if the key was completely wrong?

Reply:
IDs fixed at the SQL files.